### DISCUSSION TITLE
Welcome to Facilitating Software Architecture Discussions!

### DISCUSSION THREAD

**Discussion by andrewharmellaw** [andrewharmellaw/facilitating-software-architecture#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marcolenzo** [comment]
Hi Andrew! 👋

I discovered your book and approach at QCon London, and it felt like a friend I never knew came alongside me to help solve **one of the toughest problems I'd never cracked**.

I'm a Lead Architect and Platform Owner who, together with a wonderful group of other engineers, architects, managers, and experts, built an IaaS product (and the Platform on which it runs) from the ground up. I was literally the first person on the venture and **have always placed a strong emphasis on culture**.

We are a good example of an organization with hands-on architects who have strong and healthy relationships with their teams. One key reason is that we have one architect per team that acts as a "primus inter pares". However, the real goal has always been to have architecture done by team members.

As you say in the book, **_"traditional architectural approaches insufficiently expose teams to the option-making stage of the decision process"_**. You state such a profound truth and something I never managed to solve but only mitigated through a series of initiatives that worked partially.

Finally, I know the reason of my failures. I didn't model the decision making process properly and I've never considered the power of the advice process.

I am still not done reading the book, but I was so excited about it I had to reach out to you to tell you **"THANK YOU 🤗"**!

I'll follow up with questions once I digest everything.

**Reply by andrewharmellaw** [reply]
That makes me so happy to hear @marcolenzo. 😊  Please share any puzzles or questions you have here (in separate discussions) so the benefits of your experience can be shared for others.
